src/transactions/4_registerMultisignatureAccount.js

```diff
@@ -37,7 +42,20 @@ const registerMultisignatureAccount = ({
 	timeOffset,
 }) => {
 	const keys = cryptoModule.getKeys(passphrase);
-	const keygroupFees = keysgroup.length + 1;
+
+	const plusPrependedKeysgroup =
+		validatePublicKeys(keysgroup) && prependPlusToPublicKeys(keysgroup);
+
+	if (
+		plusPrependedKeysgroup.length === 0 ||
+		plusPrependedKeysgroup.length > 16
+	) {
+		throw new Error(
+			'You must have between 1 and 16 public keys in the keysgroup.',
+		);
+	}
```

Comment:
This could be another validation function. As above, prefer the validation -> processing -> packaging ordering.